### PR TITLE
Add march community update

### PR DESCRIPTION
**Pull request by melcorr**
Will reread Tues morning

### CODE REVIEW

File: _posts/2021-04-01-pulp-community-update-march2021.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+Version 2.11.0 is compatible with Pulpcore 3.11.
+
+The most important Debian update is that starting with Pulpcore 3.11, the MD5 checksum algorithm will be disabled by default.
+Providing MD5 checksums in repository metadata is almost universal practice within the APT ecosystem.
+For information about re-enabling MD5, or otherwise handling this change, see the new [workflow documentation](https://docs.pulpproject.org/pulp_deb/workflows/checksums.html).
+
+## Pulp Ansible updates
+
+This month, there were several minor updates Pulp Ansible with the releases of [0.5.7](https://listman.redhat.com/archives/pulp-list/2021-March/msg00002.html), [0.5.8](https://listman.redhat.com/archives/pulp-list/2021-March/msg00010.html), [0.6.2](https://listman.redhat.com/archives/pulp-list/2021-March/msg00003.html), and [0.7.1](https://listman.redhat.com/archives/pulp-list/2021-March/msg00004.html).
```
Comment (mdellweg) [inline _posts/2021-04-01-pulp-community-update-march2021.md]:
```suggestion
This month, there were several minor updates to Pulp Ansible with the releases of [0.5.7](https://listman.redhat.com/archives/pulp-list/2021-March/msg00002.html), [0.5.8](https://listman.redhat.com/archives/pulp-list/2021-March/msg00010.html), [0.6.2](https://listman.redhat.com/archives/pulp-list/2021-March/msg00003.html), and [0.7.1](https://listman.redhat.com/archives/pulp-list/2021-March/msg00004.html).
```

File: _posts/2021-04-01-pulp-community-update-march2021.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+While DevConf.cz took place at the end of February, it took a while for the videos from the sessions to be added to YouTube.
+
+If you didn't get a chance, you can watch Pavel and Daniel host a workshop on hosting your own on-premise PyPI.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yxPHEHNJwO4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Host your own on-premise Ansible Galaxy
+
+Brian also hosted a workshop where he demoed how you can host your very own on-premise Ansible Galaxy. Brian also demoed the power of the new Pulp 3 CLI.
```
Comment (mdellweg) [inline _posts/2021-04-01-pulp-community-update-march2021.md]:
```suggestion
Brian also hosted a workshop where he demoed how you can host your very own on-premise Ansible Galaxy. Along the way he demostrated the power of the new Pulp 3 CLI.
```

**Review by mdellweg — COMMENTED** [review]
Suggestions...

File: _posts/2021-04-01-pulp-community-update-march2021.md (hunk 2 above)
Comment (melcorr) [reply _posts/2021-04-01-pulp-community-update-march2021.md]: I liked this idea but "Along the way" doesn't quite work, imo, so I changed it but not to this. Thank you for the suggestion!

**Pull request merged**